base/stat.jl

```diff
@@ -170,25 +194,24 @@ stat(fd::Integer)           = stat(RawFD(fd))
 """
     stat(file)
 
-Return a structure whose fields contain information about the file.
+Return a `Base.StatStruct` whose fields contain information about the file.
```

Comment:
```suggestion
Return a structure whose fields contain information about the file.
```

The name of the structure is not a guarantee